README.md

```diff
@@ -40,6 +40,30 @@ pkg> add NiSparseArrays
 
 More to add in the next stage...
 
+## A Simple Using Case
+
+Here we present a minimal using case to illustrate how to use `NiSparseArrays` to speed up `Zygote`'s gradient. To access for more examples, navigate to `examples` directory.
+
+``` julia 
+julia> using SparseArrays, LinearAlgebra, Random, BenchmarkTools
+
+julia> A = sprand(1000, 1000, 0.1);
+
+julia> x = rand(1000);
+
+julia> using Zygote
+
+julia> @btime Zygote.gradient((A, x) -> sum(A*x), $A, $x)
+  15.065 ms (27 allocations: 8.42 MiB)
+
+julia> using NiSparseArrays
+
+julia> @btime Zygote.gradient((A, x) -> sum(A*x), $A, $x)
+  644.035 μs (32 allocations: 3.86 MiB)
+```
+
+You will see that using `NiSparseArrays` would not only speed up the computation process but also save much memory since our implementation keeps the original type rather than dense arrays in gradient computation. 
```

Comment:
```suggestion
You will see that using `NiSparseArrays` would not only speed up the computation process but also save much memory since our implementation does not convert a sparse matrix to a dense arrays in gradient computation. 
```